### PR TITLE
Add renovate security-updates-only configuration

### PR DESCRIPTION
**Pull request by dakujem**
This PR adds `renovate.json`, as suggested in https://github.com/laminas/laminas-xml/issues/12.

The file only refs the `laminas/.github` configuration.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>laminas/.github:renovate-config"
```
Comment (Ocramius) [inline renovate.json]: We want yo reference tge security -only config here

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: renovate.json (hunk 1 above)
Comment (dakujem) [reply renovate.json]: okay, done.

**Comment by internalsystemerror** [comment]
I have all these queued and waiting to push locally, but I'm still waiting on the README.md wording updates from @weierophinney and the autocloser XML from @Xerkus.

**Comment by dakujem** [comment]
I pushed the updated renovate.json that points to `renovate-config-security-updates-only`.

**Comment by internalsystemerror** [comment]
Also, It's not really as simple as just enabling renovate, as renovate relies on CI checks to tell it whether the upgrade is OK or not. So I've also been looking at getting all these security only packages to at least use PHPUnit + Psalm.

**Comment by Ocramius** [comment]
@internalsystemerror so we should hold off? 🤔

**Comment by internalsystemerror** [comment]
I'm on the fence tbh... I'm not convinced that PHPUnit alone will protect us from breaks in this repository. If you feel differently then overrule me, but I would rather we shore up the CI checks on these libs prior to enabling renovate.

Although the security only configuration doesn't automerge anything so it may be safe to do so anyway.

**Comment by Ocramius** [comment]
I'm happy with PHPUnit only here, FWIW 👍

**Review by internalsystemerror — APPROVED** [review]
Let's go with it then 👍

**Comment by dakujem** [comment]
Nice. I have no idea what I've done, but I'm glad it was what you guys wanted 😅

Thanks for maintaining the packages 💪

**Comment by internalsystemerror** [comment]
@dakujem Thanks for the contribution!

**Pull request merged**